ArtOfIllusion/src/artofillusion/SplineMeshEditorWindow.java

```diff
@@ -758,8 +758,7 @@ void subdivideCommand()
     param = new double [usize+usplitcount][vsize][numParam];
     for (i = 0; i < param.length; i++)
       for (j = 0; j < param[i].length; j++)
-        for (int k = 0; k < param[i][j].length; k++)
```

Comment:
I just realized that I did something wrong with this benchmark. This code is copying the texture parameter lists, and the object I tested on has no parameters. Basically measuring copying an empty array reference. I'll redo with real parameters. in this snippet, `k` is the number of different names parameters, correct? This means that it will tend to be a small array, and in many cases, be length '0'